src/main/java/net/minecraftforge/client/model/obj/OBJLoader.java

```diff
@@ -68,13 +68,24 @@ else if (modelLocation.getResourcePath().startsWith("models/item/"))
                     else throw e;
                 }
                 OBJModel.Parser parser = new OBJModel.Parser(resource, manager);
-                OBJModel model = parser.parse();
-                cache.put(modelLocation, model);
+                OBJModel model = null;
+                try
+                {
+                	model = parser.parse();
+                }
+                catch (OBJModel.UVsOutOfBoundsException e)
```

Comment:
This catch block is completely unnecessary, omit it and just leave the finally block